src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,22 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => {
+  if (comments.length === 0) {
+    return (
+      <>
+        <hr />
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
+  }
+
+  return (
+    <div className="CommentList">
+      {comments.map(comment => {
+        return <CommentInfo comment={comment} key={comment.id} />;
+      })}
```

Comment:
```suggestion
      {comments.map(comment => (
        <CommentInfo comment={comment} key={comment.id} />;
      ))}
```
You can immediately return with `return` keyword in an arrow function if you have only one line of code